pkg/schedule/operator/operator.go

```diff
@@ -63,6 +64,8 @@ var (
 	ExceedStoreLimit CancelReasonType = "exceed store limit"
 	// ExceedWaitLimit is the cancel reason when the operator exceeds the waiting queue limit.
 	ExceedWaitLimit CancelReasonType = "exceed wait limit"
+	// RelatedRegionID is the cancel reason when the operator is related to another region.
+	RelatedRegionID CancelReasonType = "related region id"
```

Comment:
"related region id" is not a good reason, how about something like "related merge region" or else?

Reply:
For the two operators of hot-region and other cases, maybe we also need to cancel together, so I hope to unify cancel unified reason.
what do you think?

Reply:
Yes, but it should be readable first.